Jellyfin.Server/Middleware/BaseUrlRedirectionMiddleware.cs

```diff
@@ -45,36 +45,35 @@ public async Task Invoke(HttpContext httpContext, IServerConfigurationManager se
             var localPath = httpContext.Request.Path.ToString();
             var baseUrlPrefix = serverConfigurationManager.GetNetworkConfiguration().BaseUrl;
 
-            if (!string.IsNullOrEmpty(baseUrlPrefix))
+            if (string.IsNullOrEmpty(localPath)
+                || string.Equals(localPath, "/", StringComparison.OrdinalIgnoreCase)
+                || string.Equals(localPath, "/web", StringComparison.OrdinalIgnoreCase)
+                || string.Equals(localPath, "/web/", StringComparison.OrdinalIgnoreCase)
```

Comment:
These seem very redundant given the `!localPath.StartsWith(baseUrlPrefix, StringComparison.OrdinalIgnoreCase)` check